assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -121,7 +121,9 @@ const List = styled(Area).attrs({
   }
 `;
 
-const Li = styled.li`
+const Li = styled.li.attrs({
```

Comment:
Why was this added?

Reply:
As the commit says: listbox items need to have role `option`. The carousel is a listbox.